examples/linear_model/plot_ransac.py

```diff
@@ -3,9 +3,19 @@
 Robust linear model estimation using RANSAC
 ===========================================
 
-In this example we see how to robustly fit a linear model to faulty data using
+:ref:`RANSAC <ransac_regression>` (random sample consensus) is an iterative
+algorithm to estimate parameters of a model from a dataset with outliers, especially when
+the outliers have little-to-no influence on the values of the estimates.
```

Comment:
I don't think that we need to either define what RANSAC is doing or add the reference. Instead, we can possibly cross-reference to the user guide: https://scikit-learn.org/dev/modules/linear_model.html#ransac-random-sample-consensus